charts/merlin/Chart.yaml

```diff
@@ -33,4 +33,4 @@ maintainers:
 - email: caraml-dev@caraml.dev
   name: caraml-dev
 name: merlin
-version: 0.10.21
+version: 0.10.22
```

Comment:
Given the breaking changes to the values file introduced by this chart, I think the recommendation is to use a minor version upgrade: `0.10.21` -> `0.11.0`